src/PowerSystems.jl

```diff
@@ -22,6 +22,8 @@ export Device
 export Branch
 export StaticInjection
 export ACBranch
+export ACLine
+export ACTransformer
```

Comment:
Is there such thing as a DC Transformer? .... ACBranch covers both cases already.

Reply:
but if you want all transformers... what should the abstract type be?
 - `Transformer`?

Reply:
That can be implemented but I don't see the benefit from the modeling perspective al transformers that aren't Tap or PhaseShifters are the same as an ACBranch